### DISCUSSION TITLE
Multiple Issues Created When Submitting with Image Upload

### DISCUSSION THREAD

**Discussion by akemmanuel** [community/community#87323] [Mobile] [answered]
I encountered an issue in the GitHub mobile app where, after creating an issue and uploading an image, clicking on “Submit” initially results in no response. If I press it again, the issue submission window closes, but because I pressed it multiple times, two issues are unintentionally created. This behavior seems to be linked to potential delays during image uploading.

Steps to Reproduce:

	1.	Open GitHub mobile app.
	2.	Create a new issue.
	3.	Upload an image.
	4.	Click “Submit.”
	5.	Observe the lack of immediate response.
	6.	If there’s no response, clicking “Submit” again unintentionally creates multiple issues.

Expected Behavior:
Submitting an issue with an image should not lead to the creation of multiple issues. The app should provide clear feedback during the image uploading process to avoid user confusion.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.144.0 (119864029)
Device: iPhone SE 2 (iOS 17.1.2)
</details>

**Comment by akemmanuel** [comment]
https://github.com/community/community/assets/118383304/734a0393-a902-47da-81f0-f7f54f7db790

**Reply by ghost** [reply]
Hi I guess this works wait until the image is completely uploaded. And then hit the submit. It takes time to upload and then submit maybe due to network speed.


https://github.com/community/community/assets/121353576/cba3e844-2b87-4ff4-a12c-77cb5082282d

**Reply by ghost** [reply]
So github should look into it.. .

**Reply by akemmanuel** [reply]
I clicked the submit button four times and there were four issues...

**Reply by akemmanuel** [reply]
I think the submit button isn't deactivated while uploading!

**Reply by ghost** [reply]
So yeah this is very little unnoticeable bug

**Comment by eliperkins** [comment, marked as answer]
Thanks for this bug report, @akemmanuel! This definitely looks like unexpected behavior, and I can see how frustrating having multiple issues created would be.

I've created an issue on our internal tracking repo for this bug. I'll work with the engineering team to get this addressed and follow up back here when it's been shipped. Thanks for helping us improve GitHub Mobile!

**Reply by ghost** [reply]
That's good to hear ☺️....
